tests/Feature/EventProgramTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Tests\Feature;
+
+use Aic\Hub\Foundation\Testing\FeatureTestCase as BaseTestCase;
+use App\Models\EventProgram;
+
+class EventProgramTest extends BaseTestCase
+{
+    public function test_event_programs_exist()
+    {
+        // XXX I'm not sure how useful of a test this is.
+        $this->assertDatabaseCount('event_programs', 31);
```

Comment:
Does this count all records, published or unpublished? It may be more useful to test for the presence of specific, frequently used programs. Josh could maybe give you a top 5.

Reply:
If the [list of event programs](https://github.com/art-institute-of-chicago/artic.edu/blob/fix/project-setup/database/seeders/EventProgramSeeder.php#L10-L45) I put in the seeder won't change that often, we can move that array to the `EventProgram` model. That way we can update the test to be:
- navigate to url that displays these event programs
- iterate the list to see if they all appear on the page

Reply:
To summarize what we discussed yesterday: the may not change that often, but they shouldn't require a code update to make changes or keep parity with the list in production. This is content that is managed by the CMS, and should stay changeable in that environment over time. So the array can stay in the seeder, and represent a draft list of purposes of spinning up a fresh environment.